src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs

```diff
@@ -120,6 +211,83 @@ public static void RequestPermission(string privilege)
             }
         }
 
+        /// <summary>
+        /// Triggers the permissions request for a user.
+        /// </summary>
+        /// <param name="privileges">The privacy privileges to be requested.</param>
+        /// <exception cref="ArgumentException">Thrown when an invalid parameter is passed.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when a memory error occurred.</exception>
+        /// <exception cref="System.IO.IOException">Thrown when the method failed due to an internal I/O error.</exception>
+        /// <example>
+        /// <code>
+        ///     string[] privileges = new [] {"http://tizen.org/privilege/account.read",
+        ///                                   "http://tizen.org/privilege/alarm"};
+        ///     CheckResult[] results = PrivacyPrivilegeManager.CheckPermissions(privileges);
+        ///     List&lt;string&gt; privilegesWithAskStatus = new List&lt;string&gt;();
+        ///     for (int iterator = 0; iterator &lt; results.Lenght; ++itetartor)
+        ///     {
+        ///         switch (result)
+        ///         {
+        ///             case Allow:
+        ///                 // Privilege can be used
+        ///                 break;
+        ///             case Deny:
+        ///                 // Privilege can't be used
+        ///                 break;
+        ///             case Ask:
+        ///                 // User permission request required
+        ///                 privilegesWithAskStatus.Add(privileges[iterator]);
+        ///                 break;
+        ///         }
+        ///     }
+        ///     PrivacyPrivilegeManager.RequestPermissions(privilegesWithAskStatus.ToArray());
+        /// </code>
+        /// </example>
+        /// <since_tizen> 5 </since_tizen>
+        public static void RequestPermissions(string[] privileges)
```

Comment:
Why don't you provide async method?

public static Task<RequestResponse> RequestPermissions(string[] privileges)

Reply:
And do not use array in API
Use `IEnumerable<string>` instead of `string[]`

Reply:
2. Why do you think so?
Please run below code
``` c#
            if (typeof(string[]).IsAssignableFrom(typeof(IEnumerable<string>))) {
                System.Console.WriteLine("true");
            }            
            string[] result = new string[10];
            IEnumerable<string> mylist = result;
```

Reply:
Please see .net framework design guide

X DO NOT use ArrayList or List<T> in public APIs.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections

Reply:
And 
`public static void RequestPermission(string privilege)`
It should be deprecated because it is not good usability